projects/swc-starter-webpack/src/styles.css

```diff
@@ -130,6 +130,16 @@ h3 {
     padding: 8px;
     border-radius: 3px;
 }
+#variantLabel {
+    color: white;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    background-color: rgb(193, 196, 197);
```

Comment:
any reason for these changes in this PR?

Reply:
These are just related to styling our demo test plugin.